packages/wonder-blocks-accordion/src/components/accordion-section-header.tsx

```diff
@@ -217,6 +225,10 @@ const styles = StyleSheet.create({
     disabled: {
         pointerEvents: "none",
         color: "inherit",
+
+        ":focus-visible": {
+            outline: `2px solid ${tokens.color.offBlack32}`,
+        },
     },
```

Comment:
**suggestion:** It would be great adding support to Safari v14 as well.

For reference: https://github.com/Khan/wonder-blocks/blob/0d875501f8387b6445fc9a77f753f8f0ba23c09b/packages/wonder-blocks-icon-button/src/components/icon-button-core.tsx#L333-L355